src/cache_filesystem.cpp

```diff
@@ -235,7 +282,7 @@ shared_ptr<CacheFileSystem::FileMetadata> CacheFileSystem::Stats(FileHandle &han
 	    .file_size = file_size,
 	    .last_modification_time = last_modification_time,
 	};
-	return make_shared_ptr<FileMetadata>(std::move(file_metadata));
+	return make_shared_ptr<FileMetadata>(file_metadata);
```

Comment:
I understand it's no-op change here, since `FileMetadata` is POD type;
I'm thinking of adding version tag here, which is used for [validation](https://github.com/dentiny/duck-read-cache-fs/blob/main/CHANGELOG.md#add), which is of string type.
I'm afraid we would forget in the future; clang-tidy also don't warn about move here I believe.